### DISCUSSION TITLE
Add option to assign editorial comments to users

### DISCUSSION THREAD

**Discussion by andergmartins** [publishpress/PublishPress-Planner#1245] [Feature Requests]
Assigning an editorial comment to users can force them to acknowledge the comment and do something. A checkbox is displayed for the user, so he can resolve the comment.

The user could reassign the comment (permissions: comment author, assigned user, specific capability) to another user.
The user can check and uncheck the comment as resolved.
Add filter for showing comments assigned to me or other users.
Add a column to the posts saying how many assigned (not resolved) comments there are for the post.

<img width="1001" alt="Screen Shot 2021-02-24 at 16 51 40" src="https://user-images.githubusercontent.com/407131/109057955-2a936300-76c1-11eb-8ca7-4a593ab8ed45.png">

<img width="1001" alt="Screen Shot 2021-02-24 at 16 51 40 copy" src="https://user-images.githubusercontent.com/407131/109057968-2f581700-76c1-11eb-9fda-5ed59497af94.png">

<img width="987" alt="Screen Shot 2021-02-24 at 16 58 07" src="https://user-images.githubusercontent.com/407131/109058599-f2405480-76c1-11eb-8770-439de489251e.png">

<img width="1000" alt="Screen Shot 2021-02-24 at 17 02 43" src="https://user-images.githubusercontent.com/407131/109058979-70046000-76c2-11eb-943b-3323375e10bf.png">

<img width="989" alt="Screen Shot 2021-02-24 at 17 05 22" src="https://user-images.githubusercontent.com/407131/109059130-a2ae5880-76c2-11eb-8a99-c63d879fce35.png">

Based on a ClickUp feature.

**Comment by andergmartins** [comment]
A dashboard widget could be created to display the not resolved comments, with filters.

**Comment by andergmartins** [comment]
We can make an integration to the Checklists plugin where assigned comments become a checklist item for that post. So we can add a requirement to resolve all the comments before publishing. If we remove the assigned user, the checklist item is removed as well. Marking as checked the item on the checklists will resolve the comment. Show the checkbox only for the assigned user or others with permissions for that.

**Comment by andergmartins** [comment]
If we do this, we should fix #277 first.

**Comment by andergmartins** [comment]
Add notification options for triggering notifications when we assign/resolve/unresolve/unassign a comment to someone. Add a new receiver: Assigned users on the editorial comment.

**Comment by gittothemax** [comment]
Great idea and even better implementation than I was thinking
